src/lingrex/trimming.py

```diff
@@ -0,0 +1,201 @@
+"""
+Trimming functionalities in lingrex.
+"""
+import random
+import collections
+from lingpy.sequence.sound_classes import token2class
+
+
+def revert(alms):
+    return [[row[i] for row in alms] for i in range(len(alms[0]))]        
+
+
+def get_skeleton(alms, gap="-"):
```

Comment:
After reading the paper - and also looking at the uses in the `trim_*` functions, I think this is a misnomer. The function does **not** compute a skeleton - but something like a projection of alignment sites to sound classes. Maybe `get_cv_template` would be easier to understand.

The definitions in the paper sound like "skeleton" should be reserved for the **minimal** cv templates that are required for an alignment.

Reply:
Yes, I agree with that.